Assets/General/Scripts/Editor/StoryManagerEditor.cs

```diff
@@ -0,0 +1,387 @@
+﻿/*****************************************************************************
+// Name :           StoryManagerEditor.cs
+// Author :         Charlie Polonus
+// Created :        11/7/2024
+// Description :    The custom editor for the StoryManager script
+*****************************************************************************/
+
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEditor;
+
+[CustomEditor(typeof(StoryManager))]
+public class StoryManagerEditor : Editor
+{
+    // Settings for the editor specifically
+    private bool _debugMode;
```

Comment:
Booleans should begin with words such as "is" "can" "are" etc.
This could be _isInDebugMode